### ISSUE TITLE
bilibili解析问题！！！！大神求救！急！

### ISSUE THREAD

**Issue by liuxugithub** [soimort/you-get#2067]
Please make sure these boxes are checked before submitting your issue – thank you!

- [x] You can actually watch the video in your browser or mobile application, but not download them with `you-get`.
- [x] Your `you-get` is up-to-date.
- [x] I have read <https://github.com/soimort/you-get/wiki/FAQ> and tried to do so.
- [x] The issue is not yet reported on <https://github.com/soimort/you-get/issues> or  <https://github.com/soimort/you-get/wiki/Known-Bugs>. If so, please add your comments under the existing issue.
- [x] The issue (or question) is really about `you-get`, not about some other code or project.

Run the command with the `--debug` option, and paste the full output inside the fences:

```
[PASTE IN ME]
```

If there's anything else you would like to say (e.g. in case your issue is not about downloading a specific video; it might as well be a general discussion or proposal for a new feature), fill in the box below; otherwise, you may want to post an emoji or meme instead:

> [WRITE SOMETHING]
> [OR HAVE SOME :icecream:!]

汉语翻译最终日期：2016年02月26日

在提交前，请确保您已经检查了以下内容!

- [x] 你可以在浏览器或移动端中观看视频，但不能使用`you-get`下载.
- [x] 您的`you-get`为最新版.
- [x] 我已经阅读并按 <https://github.com/soimort/you-get/wiki/FAQ> 中的指引进行了操作.
- [x] 您的问题没有在<https://github.com/soimort/you-get/issues> , <https://github.com/soimort/you-get/wiki/FAQ> 或 <https://github.com/soimort/you-get/wiki/Known-Bugs> 报告，否则请在原有issue下报告.
- [x] 本问题确实关于`you-get`, 而不是其他项目.

请使用`--debug`运行，并将输出粘贴在下面:

```
you-get -d http://www.bilibili.com/video/av10922071/
[DEBUG] get_content: http://www.bilibili.com/video/av10922071/
[DEBUG] get_content: http://interface.bilibili.com/playurl?&cid=18081460&from=miniplay&player=1&sign=09b4f6534ff0a6d3cc6547d550a681db
[DEBUG] url_info: http://ws.acgvideo.com/8/20/18081460-1.flv?wsTime=1496849768&platform=pc&wsSecret2=63e3bc0d3ef2b4348c973af0e69961c4&oi=2101741445&rate=1280
you-get: version 0.4.652, a tiny downloader that scrapes the web.
you-get: ['http://www.bilibili.com/video/av10922071/']
Traceback (most recent call last):
  File "/Users/liuxu/you-get/you-get", line 11, in <module>
    you_get.main(repo_path=_filepath)
  File "/Users/liuxu/you-get/src/you_get/__main__.py", line 92, in main
    main(**kwargs)
  File "/Users/liuxu/you-get/src/you_get/common.py", line 1422, in main
    script_main('you-get', any_download, any_download_playlist, **kwargs)
  File "/Users/liuxu/you-get/src/you_get/common.py", line 1335, in script_main
    download_main(download, download_playlist, args, playlist, output_dir=output_dir, merge=merge, info_only=info_only, json_output=json_output, caption=caption)
  File "/Users/liuxu/you-get/src/you_get/common.py", line 1151, in download_main
    download(url, **kwargs)
  File "/Users/liuxu/you-get/src/you_get/common.py", line 1415, in any_download
    m.download(url, **kwargs)
  File "/Users/liuxu/you-get/src/you_get/extractors/bilibili.py", line 181, in bilibili_download
    info_only=info_only)
  File "/Users/liuxu/you-get/src/you_get/extractors/bilibili.py", line 99, in bilibili_download_by_cid
    _, type_, temp = url_info(url)
  File "/Users/liuxu/you-get/src/you_get/common.py", line 424, in url_info
    response = urlopen_with_retry(request.Request(url))
  File "/Users/liuxu/you-get/src/you_get/common.py", line 309, in urlopen_with_retry
    return request.urlopen(*args, **kwargs)
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/urllib/request.py", line 223, in urlopen
    return opener.open(url, data, timeout)
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/urllib/request.py", line 532, in open
    response = meth(req, response)
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/urllib/request.py", line 642, in http_response
    'http', request, response, code, msg, hdrs)
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/urllib/request.py", line 564, in error
    result = self._call_chain(*args)
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/urllib/request.py", line 504, in _call_chain
    result = func(*args)
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/urllib/request.py", line 756, in http_error_302
    return self.parent.open(new, timeout=req.timeout)
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/urllib/request.py", line 532, in open
    response = meth(req, response)
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/urllib/request.py", line 642, in http_response
    'http', request, response, code, msg, hdrs)
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/urllib/request.py", line 570, in error
    return self._call_chain(*args)
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/urllib/request.py", line 504, in _call_chain
    result = func(*args)
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/urllib/request.py", line 650, in http_error_default
    raise HTTPError(req.full_url, code, msg, hdrs, fp)
urllib.error.HTTPError: HTTP Error 403: Forbidden

[在这里粘贴完整日志]
```

如果您有其他附言，例如问题只在某个视频发生，或者是一般性讨论或者提出新功能，请在下面添加;或者您可以卖个萌:

> [您的内容]

> [舔 :icecream:!]

**Comment by rosynirvana** [comment]
#2021

**Comment by shikkoku** [comment]
卧槽这么多天过去了还没合并进去吗

**Comment by soimort** [comment]
Closed via #1698.
